styleguide/derek/_partials/_footer.ejs

```diff
@@ -1,5 +1,65 @@
 <!--[footernav]-->
-<nav class="">
-
+<nav class="container footer">
+    <ul class="row clearfix">
+        <li class="col-xs-12 col-sm-6"><span class="menu-title">About Rackspace</span>
```

Comment:
So I think this may have been my fault, but I don't think this col-xs-12's are necessary.  These <li>'s span the full width by default anyway, and it will only go to 6 columns on `sm` and up.